Misc/NEWS.d/next/Library/2020-09-20-15-14-05.bpo-41810.7l8lyV.rst

```diff
@@ -0,0 +1,2 @@
+:data:`types.EllipsisType`, :data:`types.EllipsisType` and
```

Comment:
One of these should be `type.NotImplementedType`.

Reply:
Good catch; fixed in https://github.com/python/cpython/pull/22336/commits/f682623aa3001a4ac19e727c3103cdb0dce8672e.